src/Program.cs

```diff
@@ -82,6 +83,36 @@ private static void SetLoggerSettings(string configName)
 
             logger.Factory.Configuration = new XmlLoggingConfiguration(path, false);
         }
+
+        //NUR ZUM DEBUGGEN DES INSTALL-PROZESSES
+        private static void InstallTest()
+        {
+            var manager = new TaskManager();
+            var certPath = @"C:\Users\MBerthold\AppData\Roaming\senseexcel\reporting\serconnector.pem";
+            var task = manager.GetSession(new Api.SerConnection()
+            {
+                ServerUri = new Uri("http://nb-fc-208000/ser"),
+                App = "dfacdb29-6cee-4cc6-b8b1-7a89014394dd",
+                Credentials = new Api.SerCredentials()
+                {
+                    Cert = certPath,
+                    PrivateKey = @"C:\Users\MBerthold\AppData\Roaming\senseexcel\reporting\serconnector_private.key",
```

Comment:
NO private paths inside the code

Reply:
@konne
It´s only for install qmc tests. It is not active.
I hand over the paths as parameters.